src/content/en/fundamentals/getting-started/your-first-multi-screen-site/responsive.markdown

```diff
@@ -201,7 +201,7 @@ where we want to change it.
 
 600px appears to be a good place to create our first breakpoint as it will give us scope
 to reposition elements to make them fit the screen better.  We can do this
-using a technology called [Media Queries]({{site.fundamentals}}/layouts/rwd-fundamentals/#use-css-media-queries-for-responsiveness).
+using a technology called [Media Queries]({{site.fundamentals}}/design-and-ui/responsive/fundamentals/use-media-queries).
```

Comment:
Fixed! I used Pete's notation, as the /web/... syntax i used throughout the site.